crates/fig_api_client/src/clients/client.rs

```diff
@@ -97,6 +102,33 @@ impl Client {
     }
 
     pub async fn generate_recommendations(&self, input: RecommendationsInput) -> Result<RecommendationsOutput, Error> {
+        let validate_field_length = |field: &'static str, value: &str, max_length: usize| -> Result<(), Error> {
+            if value.len() > max_length {
+                return Err(Error::LengthValidation {
```

Comment:
It is recommended to just truncate the string by max_length in this case

Reply:
Fixed.